datasets/cn/national_congress/cn_national_congress.yaml

```diff
@@ -0,0 +1,44 @@
+title: China National People's Congress
+entry_point: crawler.py
+prefix: cn-npc
+coverage:
+  frequency: never 
+  start: 2024-05-16
+  end: 2028-03-01
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Current members of the Chinese National People's Congress
+description: |
+  The National People's Congress (NPC) is the highest organ of
+  state power of the People's Republic of China. It is the only 
+  branch of the Chinese government, and is unicameral. The NPC
+  is elected for a term of five years, and the last election
+  was held in 2023. Delegates to the National People's Congress
+  are allowed to hold seats in other bodies of government
+  simultaneously and the party and the NPC typically includes
+  all of the senior officials in Chinese politics.
```

Comment:
Let's add a paragraph with wikipedia's license for reuse of this data to the description

Reply:
Do you mean this part? "All text on this site is available under the terms of [a Creative Commons Attribution-Share Alike 4.0 License](https://creativecommons.org/licenses/by-sa/4.0/deed.zh)"

Reply:
yeah - so to cite it I'd say something like  "Data sourced from Wikipedia licensed [Creative Commons Attribution-Share Alike 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.zh)"